binary/main.go

```diff
@@ -15,13 +14,25 @@ import (
 )
 
 var WEB3_JWT = "WEB3_JWT"
-var UPLOAD_FILE = "temp.txt"
+var UPLOAD_FILE = "./main.go" // uploading current file itself
 
-func pinningCoreHandleFunc(address string, node pinner.PinnerNode) cid.Cid {
+func main() {
+	token, present := os.LookupEnv(WEB3_JWT)
+	if !present {
+		log.Fatalf("token (%s) not found in env", WEB3_JWT)
+	}
 
+	clientCreateReq := client.NewClientRequest(core.Web3Storage).BearerToken(token)
+	// check if cid compute true works with car uploads
+	nodeCreateReq := pinner.NewNodeRequest(clientCreateReq).CidVersion(1).CidComputeOnly(false)
+	node := pinner.NewPinnerNode(*nodeCreateReq)
 	ctx := context.Background()
+	//upload(ctx, node)
```

Comment:
we can do both. binary/main.go is just for testing purposes...to run the program and see it working. We would actually be using server/main.go really.
ADD followed by GET will now return the data (because the data is now not cleaned up from local store).